css/homepage.css

```diff
@@ -94,3 +94,14 @@ body {
 		line-height: 1.5;
 	}
 }
+
+.sr-only {
```

Comment:
If not using a library, prefer an explicit classname over a bootstrap convention: `screenreader-only`.